src/content/articles/objects-narrowed-before-usage.mdx

```diff
@@ -0,0 +1,60 @@
+---
+date: "June 1 2022"
+description: "yippee!..."
+---
```

Comment:
Todo: add a gif image like an Archer _"Can't? Or won't?"_

Reply:
Ehh, other articles aren't using them. Seems like this is a more professional blog. 🕴️